undetected_chromedriver/patcher.py

```diff
@@ -62,7 +62,17 @@ def __init__(
         prefix = "undetected"
         self.user_multi_procs = user_multi_procs
 
-        self.is_old_chromedriver = version_main and version_main <= 114
+        try:
+            # Try to convert version_main into an integer
+            version_main_int = int(version_main)
+            # check if version_main_int is less than or equal to e.g 114
+            self.is_old_chromedriver = version_main and version_main_int <= 114
+        except ValueError:
```

Comment:
```suggestion
        except (ValueError,TypeError):
```

See https://github.com/FlareSolverr/FlareSolverr/issues/962.